test/unit/mdc-select/foundation.test.js

```diff
@@ -305,9 +342,34 @@ test('#layout calls notchOutline(true) if value is not an empty string', () => {
 });
 
 test('#layout calls notchOutline(false) if value is an empty string', () => {
-  const {foundation, mockAdapter} = setupTest();
+  const {foundation} = setupTest();
   foundation.notchOutline = td.func();
-  td.when(mockAdapter.getValue()).thenReturn('');
   foundation.layout();
   td.verify(foundation.notchOutline(false), {times: 1});
 });
+
+test('#setLeadingIconAriaLabel sets the aria-label of the leading icon element', () => {
+  const {foundation, leadingIcon} = setupTest();
+  foundation.setLeadingIconAriaLabel('foo');
+  td.verify(leadingIcon.setAriaLabel('foo'), {times: 1});
+});
+
+test('#setLeadingIconContent sets the content of the leading icon element', () => {
+  const {foundation, leadingIcon} = setupTest();
+  foundation.setLeadingIconContent('foo');
+  td.verify(leadingIcon.setContent('foo'), {times: 1});
+});
+
+test('#setLeadingIconAriaLabel does nothing if the leading icon element is undefined', () => {
```

Comment:
Should these last two tests' foundation calls be wrapped in assert.doesNotThrow?